src/patternfly/components/NavSystem/docs/code.md

```diff
@@ -0,0 +1,50 @@
+## Overview
+
+The navigation system relies on several different sub-components: 
+
+* `.pf-c-nav__list` - Default navigation list. It is the basis for both default and expandable, vertical navigation.
+* `.pf-c-nav__list-simple` - Nav list simple is a shareable component this is used with grouped navigation and as secondary navigation.
+* `.pf-c-nav__list-horizontal` - Nav list horizontal is a shareable component that can be used within the page header, as primary navigation, or as tertiary navigation, when expandable, vertical navigation is implemented.
+* `.pf-c-nav__list-tertiary` - Nav list tertiary is a component that can be used within the `<main>`, as third level navigation (tertiary navigation), when expandable, vertical navigation is implemented.
+
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-label="[landmark description]"` | `pf-c-nav` |  Describes `<nav>` landmark |
```

Comment:
All the class names through the accessibility and usage sections should have `.classname` with the `.` to be consistent.

Reply:
Updated